jme3-core/src/main/resources/Common/ShaderLib/GLSLCompat.glsllib

```diff
@@ -19,18 +19,27 @@
 #endif
 
 #if __VERSION__ >= 130
+#  ifdef GL_ES
+out highp vec4 outFragColor;
+#  else
 out vec4 outFragColor;
+#endif
 #  define texture1D texture
 #  define texture2D texture
 #  define texture3D texture
 #  define textureCube texture
 #  define texture2DLod textureLod
 #  define textureCubeLod textureLod
+#  define texture2DArray texture
 #  if defined VERTEX_SHADER
 #    define varying out
 #    define attribute in
 #  elif defined FRAGMENT_SHADER
 #    define varying in
 #    define gl_FragColor outFragColor
 #  endif
-#endif
\ No newline at end of file
+#else
+#  define isnan(val) !(val<0.0||val>0.0||val==0.0)
```

Comment:
Is this needed anywhere in this pr? If not, it's probably safer to keep it out.

Reply:
isnan is used only in the water fragment shader. This is the best way I found to simulate isnan in glsl not having this function